gusto/active_tracers.py

```diff
@@ -134,15 +141,21 @@ def __init__(self, name='cloud_water', space='theta',
             transport_eqn (:class:`TransportEquationType`, optional): enumerator
                 indicating the type of transport equation to be solved (e.g.
                 advective). Defaults to `TransportEquationType.advective`.
+            density_name (str): the name of the associated density for a mixing
+                ratio when using the tracer_conservative transport. Defaults to None,
+                as this argument is not needed for other transport types.
         """
         super().__init__(f'{name}', space, variable_type,
-                         transport_eqn=transport_eqn, phase=Phases.liquid, chemical='H2O')
+                         transport_eqn=transport_eqn,
+                         density_name=density_name,
+                         phase=Phases.liquid, chemical='H2O')
 
 
 class Rain(ActiveTracer):
     """An object encoding the details of rain as a tracer."""
     def __init__(self, name='rain', space='theta',
                  variable_type=TracerVariableType.mixing_ratio,
+                 density_name=None,
```

Comment:
should the `density_name` argument come after the `transport_eqn` argument for consistency?

Reply:
Oops, good spot, thanks!